src/firefly/js/visualize/projection/ProjectionHeaderParser.js

```diff
@@ -381,21 +392,27 @@ function getFluxUnits(parse, zeroHeader) {
 function getCoordSys(params) {
     const {maptype, ctype1} = params;
 
-    if (maptype===PLATE) return EQ;
+    if (maptype===PLATE || maptype===LINEAR) return EQ;
     if (!ctype1) return -1;
 
-    const s = ctype1.substring(0, 2);
+    //LZ I have no idea what LL is, since it is there both in java and javascript, I just carry it over.
+    if(ctype1.substring(0, 2)==='LL') return EQ;
+
+    const s = ctype1.substring(0, 4);
     switch (s) {
-        case 'RA':
-        case 'DE':
-        case 'LL': return EQ;
+        case 'RA--':
+        case 'DEC-':
```

Comment:
The code never returns EQ, the return was commented out with the `LL`

Reply:
The "LL" case was before the switch statement: if(ctype1.substring(0, 2)==='LL') return EQ;

Reply:
I was trying to clean up the codes to make it more readable in some cases.  And also I try to make it inline with the standard.  I will check the changes and try to make the minimum changes possible.  Thanks!